plaso/engine/artifact_filters.py

```diff
@@ -108,7 +114,7 @@ def BuildFindSpecs(self, environment_variables=None):
           # TODO: move source.key_value_pairs iteration into
           # BuildFindSpecsFromRegistryArtifact.
           for key_path in set([
-              key_path for key_path, _ in source.key_value_pairs]):
+              key_value['key'] for key_value in source.key_value_pairs]):
```

Comment:
Why change this?

Reply:
The way we had re-worked it led to it building a set that only contained "key" in it.  For example, with:  {key: 'HKEY_LOCAL_MACHINE\System\ControlSet002\Control\Session Manager', value: 'BootExecute'}, 

key_path for key_path, _ in source.key_value_pairs]): only extracts the literal "key", whereas we want the value of "key" -- HKEY_LOCAL_MACHINE\System\ControlSet002....